sdk/attestation/azure-security-attestation/pom.xml

```diff
@@ -63,6 +63,12 @@
       <version>1.7.4</version> <!-- {x-version-update;com.azure:azure-core-test;dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
```

Comment:
>jaxb-api [](http://example.com/codeflow?start=18&length=8)

Just curious. Looking at the library - it lets you convert Java objects to Xml and vice-versa. Why use this over say Jackson that comes with azure-core?

Reply:
This library contains an API which converts `byte[]` to a hex encoded string. I am using that to validate the thumbprint of an X.509 certificate.

If Jackson comes with a hex encoder, I'll switch to that.

Reply:
NVM. I just found that Bouncycastle (which I'm already using) has a Hex encode/decode function, switching to that.